File: src/main/java/pl/yalgrin/playnite/simplesync/mapper/objects/CategoryMapper.java
package pl.yalgrin.playnite.simplesync.mapper.objects;

import org.springframework.stereotype.Component;
import pl.yalgrin.playnite.simplesync.library.domain.Category;
import pl.yalgrin.playnite.simplesync.library.dto.CategoryDTO;

@Component
public class CategoryMapper extends AbstractObjectMapper<Category, CategoryDTO> {

    @Override
    protected CategoryDTO createDTO() {
        return new CategoryDTO();
    }
}

File: src/main/kotlin/pl/yalgrin/playnite/simplesync/library/mapper/CategoryMapper.kt
package pl.yalgrin.playnite.simplesync.library.mapper

import org.springframework.stereotype.Component
import pl.yalgrin.playnite.simplesync.library.domain.Category
import pl.yalgrin.playnite.simplesync.library.dto.CategoryDTO

@Component
class CategoryMapper : LibraryObjectMapperImpl<Category, CategoryDTO>() {
    override fun createDTO(): CategoryDTO = CategoryDTO()
}